### ISSUE TITLE
Subscription not active

### ISSUE THREAD

**Issue by jainavana** [jitpack/jitpack.io#4160]
Please provide:

- Link to build log from https://jitpack.io

https://jitpack.io/com/github/navana-tech/cva_lib_app/Version_1.0.501/cva_lib_app-Version_1.0.501.pom

- Does the project build on your machine with **the same** commands (e.g. ./gradlew install) ?
No
- What error are you seeing?

```
Could not resolve com.github.navana-tech:cva_lib_app:Version_1.0.501.
         > Could not get resource 'https://jitpack.io/com/github/navana-tech/cva_lib_app/Version_1.0.501/cva_lib_app-Version_1.0.501.pom'.
            > Could not GET 'https://jitpack.io/com/github/navana-tech/cva_lib_app/Version_1.0.501/cva_lib_app-Version_1.0.501.pom'. Received status code 403 from server: Forbidden
```

Thank you!

**Comment by jainavana** [comment]
This is now solved

**Comment by jainavana** [comment]
Solved by closing and reopening my jitpack account

**Comment by kinshuk-zupee** [comment]
Please provide:

Link to build log from https://jitpack.io
https://www.jitpack.io/com/github/navana-tech/cva_lib_app/1.0.81/cva_lib_app-1.0.81.pom'

Does the project build on your machine with the same commands (e.g. ./gradlew install) ?
No

What error are you seeing?
Could not GET 'https://www.jitpack.io/com/github/navana-tech/cva_lib_app/1.0.81/cva_lib_app-1.0.81.pom'. Received status code 403 from server: Forbidden

**Comment by jainavana** [comment]
Thanks for letting us know @kinshuk-zupee. We have resolved this issue and have tested this version is able to be retrieved appropriately. Could you confirm?

**Comment by kinshuk-zupee** [comment]
It is working now. Thank you for your prompt reply @jainavana

**Comment by hiteshdroid** [comment]
I am facing the same issue @jainavana Can you help me with the resolution. 

Could not GET 'https://www.jitpack.io/com/github/navana-tech/cva_lib_app/1.0.81/cva_lib_app-1.0.81.pom'. Received status code 403 from server: Forbidden

**Comment by navanatech** [comment]
Sorry for the issue @hiteshdroid! We have just resolved this issue and confirmed ability to pull this version successfully. Could you please confirm.

Btw, we have since released a 2.0 version of this product. Linking you to the [docs](https://zabaan-docs.navanatech.in/1.2.2/), [maven](https://search.maven.org/artifact/in.navanatech/zabaan-sdk/1.2.2/aar) and new [cms](https://zabaan-cms.navanatech.in/). 

Latest iteration of this product has a bunch of really cool features like:

1. 70% faster initialisation and play times via a priority downloader that downloads audio files when required.
2. Written in Kotlin and very stable
3. Connected to a much more mature, stable and user friendly [CMS](https://zabaan-cms.navanatech.in/) (feel free to create an account and check it out). 

Would be happy to assist you guys with a migration.

**Comment by jainavana** [comment]
Hi @hiteshdroid, is this resolved?
